### PR TITLE
words: flappy birds, but real life

### PR DESCRIPTION
**Pull request by catzkorn**
These birds have been visually identified as having tails. Science prevails.

### CODE REVIEW

File: words/tails.txt [hunk 1]
```diff
@@ -577,3 +577,8 @@ raven
 corvus
 mellori
 bat
+sapsucker
+woodpecker
+nuthatch
```
Comment (maisem) [inline words/tails.txt:582]: Meta question should we sort them at some point?

Comment (catzkorn) [reply words/tails.txt:582]: The chaos is all a part of the fun

Comment (creachadair) [reply words/tails.txt:582]: They are sorted—by time of addition 😆

**Review by creachadair — APPROVED** [review]

File: words/tails.txt (hunk 1 above)
Comment (catzkorn) [reply words/tails.txt:582]: Really, it's largely a timeline of what im thinking and when. This week? I'm with the birds. Next week, maybe I'll be a musical scale 🤷🏻‍♀️I like to live on the edge.

**Pull request merged**